docs/design/data/armor.yaml

```
@@ -0,0 +1,597 @@
+source: docs/_incoming/XM Design Bible - Armor.tsv
+note: This file mirrors the complete Armor TSV to ensure full fidelity. Slot-sharded files can be added separately if needed.
+tsv: |
+  Type	Level	Item Name	Source	DEF	HP	EP	ACC	ATK	EVA	MGT	PER	VIT	WIL	AGI	SOC	BST	BRW	ELM	HNT	KPR	MDR	NGT	TEC
+  Shield	1	Verdant Guard	Armorcraft	1										x				x	x		
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)